sql-to-dbsp-compiler/lib/sqllib/src/interval.rs

```diff
@@ -103,16 +112,24 @@ deserialize_without_context!(ShortInterval);
 #[archive(compare(PartialEq, PartialOrd))]
 #[serde(transparent)]
 pub struct LongInterval {
-    days: i32,
+    months: i32,
```

Comment:
not sure what's going on here

Reply:
This actually was a bug which I am fixing now. I will need to add more tests for it.